app/src/main/java/comcmput301f16t01/github/carrier/MockFareCalculator.java

```diff
@@ -11,10 +11,10 @@
      * @param end
      */
     public MockFareCalculator(Location start, Location end) {
-        super(start, end);
+        //super(start, end);
```

Comment:
You made this, haha. I think when you and Kieter combined your work it went awry. 
Is calling super() what you want there?